src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -684,7 +693,7 @@ describe("Shipping", () => {
           userEvent.type(name, "First Last")
           userEvent.clear(name)
 
-          expect(screen.getByText("This field is required")).toBeInTheDocument()
+          expect(screen.getByText(/[\w\s]is required/)).toBeInTheDocument()
```

Comment:
This is another example where I'd love to assert the customized error message. Only "Full name is required" should be in the document, so this would mask failures if "Postal code is required" is in the document instead.